lang/py/pyproject.toml

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+[build-system]
+requires = ["setuptools>=57", "wheel"]  # PEP 508 specifications.
```

Comment:
Before you merge this, consider the changes in #1239. I wasn't aware of the AVRO-3189 issue at the time, but I think it addresses it and improves the packaging at the same time.

Reply:
OK!  I missed your earlier JIRA and earlier PR, I'll point the JIRA to your better PR.  I appreciate the attention that  you give to proper packaging :D   It's such a meticulous art.